src/ophys_etl/transforms/convert_rois.py

```diff
@@ -0,0 +1,193 @@
+    id = Int(required=True,
+             description=("Unique ID of the ROI, get's overwritten writting "
+                          "to LIMS"))
+    x = Int(required=True,
+            description="X location of top left corner of ROI in pixels")
+    y = Int(required=True,
+            description="Y location of top left corner of ROI in pixels")
+    width = Int(required=True,
+                description="Width of the ROI in pixels")
+    height = Int(required=True,
+                 description="Height of the ROI in pixels")
+    valid_roi = Bool(required=True,
+                     description=("Boolean indicating if the ROI is a valid "
+                                  "cell or not"))
+    mask_matrix = List(List(Bool), required=True,
+                       description=("Bool nested list describing which pixels "
+                                    "in the ROI area are part of the cell"))
+    max_correction_up = Float(required=True,
+                              description=("Max correction in pixels in the "
+                                           "up direction"))
+    max_correction_down = Float(required=True,
+                                description=("Max correction in pixels in the "
+                                             "down direction"))
+    max_correction_left = Float(required=True,
+                                description=("Max correction in pixels in the "
+                                             "left direction"))
+    mask_image_plane = Int(required=True,
+                           description=("What tiff file the ROI lie "
+                                        "within. this is being deprecated so "
+                                        "will always be assigned 0"))
+    exclusion_labels = List(Int, required=True,
+                            description=("Codes for reasoning of exclusion of "
+                                         "an ROI"))
+
+
+class BinarizeAndCreateROIsOutputSchema(ArgSchema):
+    old_rois = Nested(OldSegmentationROISchema, many=True)
+
+
+class BinarizerAndROICreator(ArgSchemaParser):
+    default_schema = BinarizeAndCreateROIsInputSchema
+
+    def binarize_and_create(self):
+
+        self.logger.name = type(self).__name__
+        self.logger.setLevel(self.args.pop('log_level'))
+
+        # load in the rois from the stat file and movie path for shape
+        self.logger.info("Loading suite2p ROIs and video size.")
+        suite2p_stats = np.load(self.args['suite2p_stat_path'],
+                                allow_pickle=True)
+        with h5py.File(self.args['motion_corrected_video'], 'r') as open_vid:
+            movie_shape = open_vid['data'][0].shape
+
+        # binarize the masks
+        self.logger.info("Binarizing the ROIs created by Suite2p.")
+        coo_rois = suite2p_rois_to_coo(suite2p_stats, movie_shape)
+
+        binarized_coo_rois = []
+        for coo_roi in coo_rois:
+            if self.args['binary_quantile'] is not None:
```

Comment:
This can be simplified and removed I think if the `binary_quantile` field can only be a `Float` instead of float/None.

Reply:
Removed the None type and removed logic to handle none type